docs/UserGuide/API/Time-zone.md

```diff
@@ -0,0 +1,78 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+    
+        http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# Time zone
+
+When a client connects to the IoTDB server, it can specify the time zone to be used for this connection. If not specified, the default time zone is the one of the client.
+
+The time zone can be set in both JDBC and session native interface connections. The usage is as follows:
+
+```java
+JDBC: (IoTDBConnection) connection.setTimeZone("+08:00");
+
+Session: session.setTimeZone("+08:00");
+```
```

Comment:
Should we tell users the time zone usage in CLI here?
For example, we can set the time zone by the following statement.
```sql
set time_zone=+04:00
```

Reply:
All right.